server/models/contests/getModeratorContests.js

```diff
@@ -11,7 +11,7 @@ function getModeratorContests({ username, query }) {
   return new Promise((resolve, reject) => {
     const limit = Number(query.limit)
     const cursor = Number(query.cursor)
-    let sqlQuery = `SELECT id,name,creator,start_time,end_time,participants_count,public,confidential_questions FROM contests c JOIN contests_moderators cm ON c.id=cm.contest_id WHERE cm.moderator=? `
+    let sqlQuery = `SELECT c.id,name,creator,start_time,end_time,participants_count,public,confidential_questions FROM contests c JOIN contests_moderators cm ON c.id=cm.contest_id WHERE cm.moderator=? `
```

Comment:
Actually, it is required because the earlier query was giving the following error when I ran it in MySQL workbench:
`Column 'id' in field list ambiguous`

Actually, it is a common error that occurs when we perform joins. Here, `id` column is present in both `contests` table as well as `contests_moderators` table, so MySQL gets confused whose `id` column, I need to return ?